.ci/xenial.jenkinsfile

```diff
@@ -56,6 +66,16 @@ node('Linux') {
         job_environment['DOCKER_TAG'] = "${branchName}-${commit}"
         //job_environment['BUILDKIT_INLINE_CACHE'] = '1'  // This makes the image usable as cache source
         echo "job_environment:\n\n${ job_environment.collect({ k, v -> "${k }=${v }"}).join('\n')}"
+
+        // Parse more information
+        distro = job_environment['DISTRO']
+        conanVersion = job_environment['CONAN_VERSION']
+        originalUsername = originalEnvironment['DOCKER_USERNAME']
+
+        echo "gccVersions: ${gccVersions.join(', ')}"
+        echo "clangVersions: ${clangVersions.join(', ')}"
+        assert gccVersions.size() > 0: "Expecting to build some GCC versions at least"
+        assert clangVersions.size() > 0: "Expecting to build some Clang versions at least"
```

Comment:
What does happen if we want to build only gcc or clang?

Reply:
Currently, it is not possible, the CI will always build both of them. If we remove GCC (or Clang) from the repo, then we should remove them from the CI as well.

Reply:
TBH, I'm thinking about creating a generic job to build all the images and call it from the multibranch one... it will allow us to manually configure the images and build them (everything could be configurable: conan, python, cmake,... gcc-versions, clang-versions,...), but it is something for the future.